.openshift-ci/post_tests.py

```diff
@@ -75,7 +76,7 @@ def __init__(
         self.artifact_destination_prefix = artifact_destination_prefix
         self.data_to_store = []
 
-    def run(self, test_outputs=None):
+    def run(self, test_outputs=None, test_results=None):
```

Comment:
This may be a bit confusing, since it does process `test_outputs` but not `test_results`. I can think of two ways:
- barf if `test_results is not None` (better fail loud than silent), or
- move the functionality to this class